integration-tests/tests/specs/features/search/lineage-field.spec.ts

```diff
@@ -7,7 +7,7 @@ import { v4 as uuidv4 } from 'uuid';
 const SEQUENCE = 'ATTGATCTCATCATTT';
 
 test('Lineage field lineage counts', async ({ page, pageWithGroup }) => {
-    test.setTimeout(95_000);
+    test.setTimeout(900_000);
```

Comment:
this increase doesn't have an affect - weirdly I can do these steps without issues on the preview so I'm not sure why the intergration tests are failing